rotkehlchen/api/v1/schemas.py

```diff
@@ -3531,3 +3532,55 @@ class FalsePositveSpamTokenSchema(Schema):
 
 class SpamTokenSchema(Schema):
     token = AssetField(expected_type=EvmToken, required=True)
+
+
+class CalendarCommonEntrySchema(Schema):
+    name = fields.String(required=True)
+    description = fields.String(load_default='')
```

Comment:
Aha so if missing you have it empty? Isn't it better to just keep `None` for consistency with the DB schema also? It's optional and can be `NULL` there